package/tests/test_api/test_graphql.py

```diff
@@ -513,3 +520,16 @@ def test_graphql_runs_tracking_data_endpoint(
             }
         }
         assert response.json() == expected_response
+
+
+class TestGraphQLSchema:
+    def testApolloSchema(self):
+        here = path.abspath(
+            path.join(path.dirname(__file__), "..", "..", "..", "src", "apollo")
+        )
+        changes = []
+        with open(path.join(here, "schema.graphql")) as data:
```

Comment:
This could be improved with pathlib. Something like

```python
schema_file = Path(__file__).parents[3] / "src" / "apollo" / "schema.graphql")

with schema_file.open() as f:
    ... 
```